v2/azure-arm.yaml

```diff
@@ -1164,6 +1164,9 @@ objectModelConfiguration:
       PrivateDnsZone:
         $export: true
         $supportedFrom: v2.0.0-beta.2
+      PrivateDnsZones_VirtualNetworkLink:
```

Comment:
Consider exporting `2020-06-01` `PrivateDnsZones_VirtualNetworkLink` instead? I'm not sure what the difference between the two API versions is but you don't have to match the parents API version for the child resource. Should probably use the latest version if we can.